miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -444,6 +450,19 @@ open class MiniAppMessageBridge {
         if (this::screenBridgeDispatcher.isInitialized)
             screenBridgeDispatcher.allowScreenOrientation = allowScreenOrientation
     }
+
+    @SuppressWarnings("SwallowedException")
+    private fun onSetCloseAlert(callbackId: String, jsonStr: String) = try {
+        val callbackObj = Gson().fromJson(jsonStr, CloseAlertInfoCallbackObj::class.java)
+        val alertInfo = callbackObj.param.closeAlertInfo
+        onMiniAppShouldClose(alertInfo)
```

Comment:
@khairul-alam-licon I think we can remove this `onMIniAppShouldClose` and set the value here and change the function name `onSetCloseAlert` to ``onMIniAppShouldClose`.

Reply:
Yeah, I have updated the changes.